### PR TITLE
All revisions API  added.

### PR DESCRIPTION
**Pull request by Abhishekism9450**
All revisions API  added.

I have also added the sample code in "Media Action API: Allrevisions" .
https://www.mediawiki.org/wiki/API:Allrevisions

### CODE REVIEW

**Comment by srish** [comment]
@Abhishekism9450 Hi! Please don't add any sample code to documentation on MediaWiki.org in the future before getting it reviewed here in this repo.

File: modules.json [hunk 1]
```diff
@@ -367,4 +367,16 @@
...
     	}
     }
+    {
+        "filename": "get_allrevisions.py",
+        "docstring": "Demo of `AllRevisions` module: List All Revisions of all pages and all users.",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params":{
+    		"action": "query",
+    		"format": "json",
+    		"list": "allrevisions",
```
Comment (srish) [inline modules.json]: This example should show how to make a get request to fetch a list of all revisions by user `Place holder` on the English Wikipedia. Add any necessary parameters that would be required for this.

File: python/get_allrevisions.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+    get_allpages.py
```
Comment (srish) [inline python/get_allrevisions.py]: File name and description here are different than what you've added in `modules.json` file. It looks like didn't autogenarate this file..

**Comment by NikitaRana07** [comment]
@srish Since Abhishek hasn't responded in a while I'm guessing he might be applying for GSOC. Can I work on this and update the documentation in the format of the template.

**Comment by srish** [comment]
@NikitaRana07 Please be patient! :) Let's give @Abhishekism9450 some more time to respond on this thread.

**Comment by NikitaRana07** [comment]
Yeah, Totally! :)

**Comment by NikitaRana07** [comment]
@srish It has been 13 days since this pull request and 4 days since my comment, I even tried to contact 
 him on zulip but got no reply. what now?

**Comment by Abhishekism9450** [comment]
@srish @NikitaRana07  sorry for the delay actually i was having some issues. 
I am participating in GSoc , so Nikita you can take my task.

**Comment by NikitaRana07** [comment]
@Abhishekism9450 Thanks for the info! @srish Can I work on this?

**Pull request closed without merging**